docs/appendices/release-notes/4.2.4.rst

```diff
@@ -0,0 +1,59 @@
+.. _version_4.2.4:
+
+=============
+Version 4.2.4
+=============
+
+Released on 2020-08-26.
+
+.. NOTE::
+
+    If upgrading a cluster, you must be running CrateDB 4.0.2 or higher before
+    you upgrade to 4.2.4.
+
+    We recommend that you upgrade to the latest 4.1 release before moving to
+    4.2.4.
+
+    A rolling upgrade from 4.1.7+ to 4.2.4 is supported.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+
+See the :ref:`version_4.2.0` release notes for a full list of changes in the
+4.2 series.
+
+
+Fixes
+=====
+
+- Fixed an issue that could lead to a ``IndexOutOfBoundsException`` when using
+  virtual tables and joins.
+
+- Fixed an issue that declared the rule optimizer settings as global. The
+  settings are now session local.
+
+- Fixed an issue that prevented the ``MATCH`` predicate from working in mixed
+  clusters running 4.1.8 and 4.2.
+
+- Fixed an issue that prevented user defined functions in a custom schema from
+  working if used in a generated column expression.
+
+- Fixed an issue that allowed users to use a function in a generated column
+  that didn't fully match the given arguments, leading to a subsequent runtime
+  failure when trying to access tables.
+
```

Comment:
Looks like the following fix on master was not backported:

> - Fixed exposure of the full qualified name of a sub-script column in
  `information_schema.tables.partitioned_by` and
  `pg_catalog.pg_attribute.attname` to use the CrateDB SQL compatible identifier.


Could you check whether that was intentional?

Reply:
I couldn't find any information on the original pr https://github.com/crate/crate/pull/10413 if this was intentional. Shall I backport it and add it to the 4.2.4 release ?

Reply:
I don't see a reason why it shouldn't be backported, so yes please do so.